apps/ledger-live-mobile/e2e/page/trade/stake.page.ts

```diff
@@ -8,45 +8,56 @@ export default class StakePage {
   cosmosAssestsRemainingId = "cosmos-assets-remaining";
   cosmosDelegatedRatioId = (delegatedPercent: number) => `delegate-ratio-${delegatedPercent}%`;
   cosmosAllAssestsUsedText = "cosmos-all-assets-used-text";
-  summaryContinueButtonId = "cosmos-summary-continue-button";
+  cosmosSummaryContinueButtonId = "cosmos-summary-continue-button";
+  cosmosDelegationStartId = "cosmos-delegation-start-button";
+  cosmosDelegationAmountContinueId = "cosmos-delegation-amount-continue";
+  currencyRow = (currencyId: string) => `currency-row-${currencyId}`;
+  zeroAssetText = "0\u00a0ATOM";
 
   async selectCurrency(currencyId: string) {
-    const id = "currency-row-" + currencyId;
-    await waitForElementById(id);
-    await tapById(id);
-  }
-
-  async selectAccount(accountId: string) {
-    const id = "account-card-" + accountId;
+    const id = this.currencyRow(currencyId);
     await waitForElementById(id);
     await tapById(id);
   }
 
   async delegationStart() {
```

Comment:
this method is not in use anywhere, do we need to keep it anyway?

Reply:
It's part of a path that should be tested. (No prior delegation in account)